docs/using-the-python-driver/using-plugins/UsingTheLimitlessConnectionPlugin.md

```diff
@@ -0,0 +1,49 @@
+| `limitless_transaction_router_monitor_disposal_time_ms` | Integer |    No    | This property is the time in milliseconds that a Limitless transaction router monitor can remain unused before it is disposed. This ensures that in periods of long inactivity, the database isn't being needlessly polled and the resources associated with the monitor can be cleaned up. Note that when a new connection is created, a new Limitless transaction router monitor will also be created to resume polling the database.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         | `600000`      | `300000`      |
+| `limitless_connection_max_retries_ms`                   | Integer |    No    | This property is the max number of retries the Limitless Connection Plugin will attempt when failing to connect to the database. During these retries, the plugin will attempt to connect to the least loaded transaction router that is available. If the max number of connection retries is exceeded, then the plugin will throw a `SQLException`. In this scenario, it is likely that the database is in an unhealthy state, and the `SQLException` should be caught and handled by your application.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | `5`           | `13`          |
```

Comment:
We will need to replace the occurrences of `SQLException` with `AwsWrapperError`